app/serializers/sprangular/user_serializer.rb

```diff
@@ -1,24 +1,14 @@
 module Sprangular
   class UserSerializer < BaseSerializer
-    attributes *user_attributes
+    attributes *(user_attributes | [:bill_address_id, :ship_address_id])
 
-    has_many :completed_orders, serializer: Sprangular::LiteOrderSerializer
-    has_many :addresses, serializer: Sprangular::AddressSerializer
-    has_many :payment_sources, serializer: Sprangular::PaymentSourceSerializer
-    has_one :order, serializer: Sprangular::OrderSerializer
-    has_one :shipping_address, serializer: Sprangular::AddressSerializer
-    has_one :billing_address, serializer: Sprangular::AddressSerializer
-
-    def completed_orders
-      object.orders.complete
-    end
+    has_many :addresses,        serializer: Sprangular::AddressSerializer
+    has_many :payment_sources,  serializer: Sprangular::PaymentSourceSerializer
+    has_one  :bill_address,     serializer: Sprangular::AddressSerializer
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.